File: webapp-java-spring/src/main/java/de/ehmke/entity/Appointment.java
package de.ehmke.entity;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "appointment")
public class Appointment
{

    /**
     * The id
     */
    @Id
    @Column(name = "id")
    private Long id;

    /**
     * The when date
     */
    @Column(nullable = false, name = "when_date")
    private Date whenDate;

    /**
     * The what
     */
    @Column(nullable = false, name = "what")
    private String what;

    /**
     * The calendar
     */
    @ManyToOne()
    @JoinColumn(nullable = false, name = "calendar_id")
    private Calendar calendar;

    /**
     * @return the id
     */
    public Long getId()
    {
        return this.id;
    }

    /**
     * @param id the id
     */
    public void setId(Long id)
    {
        this.id = id;
    }

    /**
     * @return the when date
     */
    public Date getWhenDate()
    {
        return whenDate;
    }

    /**
     * @param whenDate the when date
     */
    public void setWhenDate(Date whenDate)
    {
        this.whenDate = whenDate;
    }

    /**
     * @return the what.
     */
    public String getWhat()
    {
        return what;
    }

    /**
     * @param what the what
     */
    public void setWhat(String what)
    {
        this.what = what;
    }

    /**
     * @return the calendar
     */
    public Calendar getCalendar()
    {
        return calendar;
    }

    /**
     * @param calendar the calendar
     */
    public void setCalendar(Calendar calendar)
    {
        this.calendar = calendar;
    }
}

File: webapp-php-symfony/src/AppBundle/Entity/Appointment.php
<?php
namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Appointment
{
    /**
     * @ORM\Id
     * @ORM\Column(type="bigint")
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\Column(name="when_date", type="datetime", nullable=false)
     */
    private $when;

    /**
     * @ORM\Column(type="text", nullable=false)
     */
    private $what;

    /**
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Calendar", inversedBy="appointments")
     * @ORM\JoinColumn(name="calendar_id", referencedColumnName="id")
     */
    private $calendar;

    /**
     * Get id
     *
     * @return integer
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set when
     *
     * @param \DateTime $when
     * @return Appointment
     */
    public function setWhen($when)
    {
        $this->when = $when;

        return $this;
    }

    /**
     * Get when
     *
     * @return \DateTime
     */
    public function getWhen()
    {
        return $this->when;
    }

    /**
     * Set what
     *
     * @param string $what
     * @return Appointment
     */
    public function setWhat($what)
    {
        $this->what = $what;

        return $this;
    }

    /**
     * Get what
     *
     * @return string
     */
    public function getWhat()
    {
        return $this->what;
    }

    /**
     * Set calendar
     *
     * @param Calendar $calendar
     * @return Appointment
     */
    public function setCalendar(Calendar $calendar = null)
    {
        $this->calendar = $calendar;

        return $this;
    }

    /**
     * Get calendar
     *
     * @return Calendar
     */
    public function getCalendar()
    {
        return $this->calendar;
    }
}
